docs/installation.rst

```diff
@@ -0,0 +1,46 @@
+Installation
+============
+
+Setup profile configuration
+---------------------------
+
+Before using this tool, you need to setup the AWS configuration and credential files. Follow
+`CLI configuration documentation`_ to create these two files.
+
+Example config file:
+
+.. code-block::
+
+    $ cat ~/.aws/config
+    [default]
+    region = us-east-1
+
+Example credential file:
+
+.. code-block::
+
+   $ cat ~/.aws/credentials
+   [default]
+   aws_secret_access_key = <YOUR SECRET ACCESS KEY>
+   aws_access_key_id = <YOUR ACCESS KEY ID>
+
+Install awspub using snap
+-------------------------
+
+``awspub`` is available in the `Snapstore`_, and it can be installed using:
```

Comment:
Let's use the same formatting of awspub throughout i.e. either `awspub` or ``awspub``